exercises/protein-translation/protein_translation_test.py

```diff
@@ -1,58 +1,69 @@
 import unittest
 
-from protein_translation import of_codon, of_rna
+from protein_translation import proteins
+
+# Tests adapted from problem-specifications/canonical-data.json @ v1.0.0
```

Comment:
For consistency with other exercises, can you please change the spacing around this line? Should be 2 blank lines above, 1 below. (In the back of my mind I have a feeling I may have already said this, but switched the numbers; if so, I apologize for the misinformation)